js/g-cal.js

```diff
@@ -0,0 +1,229 @@
+var CLIENT_ID = 'CLIENT_ID';
```

Comment:
Nope, just a place holder in case retrieving the key from the json file didn't work. I needed something to see in the console in case it failed. 

The file is all me with the exception of the 6 or lines used for connecting to the Google API, that comes from the official Google Developers quickstart for the Calendar API.